test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -304,8 +304,7 @@ static public void testCreateArrayPoint2D() throws Throwable {
 		assertEquals(getY.invoke(point2D_2_check), getY.invoke(point2D_2));
 	}
 
-	/* https://github.com/eclipse-openj9/openj9/issues/19692 tracks excluded tests */
```

Comment:
There is another test disabled with a comment mentioning issue 19692: https://github.com/eclipse-openj9/openj9/blob/677181c59cd63396b455fb00927a3b63365a4115/test/functional/Valhalla/src/org/openj9/test/lworld/ValhallaAttributeTests.java#L164